pom.xml

```diff
@@ -94,13 +102,23 @@ THE SOFTWARE.
             <url>https://repo.jenkins-ci.org/public/</url>
         </pluginRepository>
     </pluginRepositories>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
+                <artifactId>script-security</artifactId>
+                <version>1.25</version>
+            </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins.icon-shim</groupId>
+                <artifactId>icon-set</artifactId>
+                <version>2.0.3</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
```

Comment:
Fixes the following Maven Enforcer errors after upgrading to the latest plugin parent POM:

```
[WARNING] Rule 4: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for org.jenkins-ci.plugins:script-security:1.13 paths to dependency are:
+-org.jenkins-ci.plugins:throttle-concurrents:2.0.2-SNAPSHOT
  +-org.jenkins-ci.plugins:matrix-project:1.8
    +-org.jenkins-ci.plugins:script-security:1.13
and
+-org.jenkins-ci.plugins:throttle-concurrents:2.0.2-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-durable-task-step:2.8
    +-org.jenkins-ci.plugins:script-security:1.25
and
+-org.jenkins-ci.plugins:throttle-concurrents:2.0.2-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-cps:2.28
    +-org.jenkins-ci.plugins:script-security:1.24
and
+-org.jenkins-ci.plugins:throttle-concurrents:2.0.2-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-cps:2.28
    +-org.jenkins-ci.plugins:script-security:1.24
and
+-org.jenkins-ci.plugins:throttle-concurrents:2.0.2-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-support:2.13
    +-org.jenkins-ci.plugins:script-security:1.21
and
+-org.jenkins-ci.plugins:throttle-concurrents:2.0.2-SNAPSHOT
  +-org.jenkins-ci.plugins.workflow:workflow-support:2.13
    +-org.jenkins-ci.plugins:script-security:1.21
, 
Require upper bound dependencies error for org.jenkins-ci.plugins.icon-shim:icon-set:1.0.5 paths to dependency are:
+-org.jenkins-ci.plugins:throttle-concurrents:2.0.2-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-core:2.60.3
    +-org.jenkins-ci.plugins.icon-shim:icon-set:1.0.5
and
+-org.jenkins-ci.plugins:throttle-concurrents:2.0.2-SNAPSHOT
  +-org.jenkins-ci.plugins:matrix-auth:1.4
    +-org.jenkins-ci.plugins.icon-shim:icon-shim:2.0.3
      +-org.jenkins-ci.plugins.icon-shim:icon-set:2.0.3
```